protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/job/JobBatchRecord.java

```diff
@@ -125,6 +126,12 @@ public boolean isTruncated() {
     return truncatedProp.getValue();
   }
 
+  @Override
+  public List<String> getTenantIds() {
+    // todo(#13318): replace dummy implementation
+    return List.of();
```

Comment:
Interesting point. Since this is temporary code, it doesn't matter, and I'll leave the code untouched. But we can still discuss it. 

💭 I can see that `Collections.emptyList` is more explicit, but List.of() is also clearly an empty list. Many languages have `[]` as an empty list literal which and that's also readable. It makes me wonder why the maintainers added `List.of` if they didn't intend for it to replace `Collections.emptyList` (note that both produce an immutable empty list). IMO verbose does not necessarily mean more readable. So my preference stays in the usage of `List.of()`, but I wouldn't reject pull requests using `Collections.emptyList`.

@koevskinikola Does this change your perspective? If not, can you clarify why you'd prefer `Collections.emptyList`?

Reply:
I agree with your decision, I just wrote down my thoughts in case you'd like to discuss this.

I don't really have a preference except for readability, and as this is temporary code, it's fine as it is. I was mainly guiding myself on [this comment](https://stackoverflow.com/a/46619739), and it seems that `List.of()` is a newer (Java 9+) concept. All the other pro/con arguments there aren't applicable.

Reply:
Interesting read. The serializability should never be an issue, but the `contains(null` behavior is quite different. Luckily that's just an edge case, but again this makes me wonder why the JDK maintainers chose this behavior 🤷 They must have had their reasons

Reply:
💡 @koevskinikola [This SO answer](https://stackoverflow.com/a/46608175/3442860) mentions that null hostility started with java 1.5 and that supporting null in lists is a leftover of the old (1.2) collections framework

Reply:
💡 It's a deliberate design decision: https://youtu.be/ogRVWXuuAU4?t=1341